### DISCUSSION TITLE
Why don't O_MBAR plots have 1s on the diagonals?

### DISCUSSION THREAD

**Discussion by StefanIvanovIBBR** [alchemistry/alchemlyb#333] [General]
The overlap matrices produced by Alchemlyb don't have 1s on the diagonal. 

In the documentation page, for example, we have the following code and plot:

https://alchemlyb.readthedocs.io/en/latest/visualisation.html

where the overlap between state 0 and state 0 is... 0.49. Why is this?

**Comment by orbeckst** [comment]
See 

Pavel V Klimovich, Michael R Shirts, and David L Mobley. Guidelines for the analysis of free energy calculations. J Comput Aided Mol Des, 29(5):397–411, May 2015. doi: [10.1007/s10822-015-9840-9](https://doi.org/10.1007/s10822-015-9840-9).

The rows and columns add up to 1. Think of it as the fraction of i that overlaps with j.
